pkg/local_object_storage/writecache/persist.go

```diff
@@ -26,7 +26,7 @@ func (c *cache) persistLoop() {
 			sort.Slice(m, func(i, j int) bool { return m[i].addr < m[j].addr })
 
 			start := time.Now()
-			c.persistObjects(m)
+			c.persistSmallObjects(m)
```

Comment:
Is it okay to persist only small objects there?

Reply:
Yes, becaues of https://github.com/nspcc-dev/neofs-node/blob/master/pkg/local_object_storage/writecache/put.go#L33